src/components/transaction-log/transaction-log.component.ts

```diff
@@ -1,12 +1,34 @@
-import { Component } from "@angular/core";
+import { Component, Input } from "@angular/core";
+import { HomePage } from "../../pages/home/home";
 
 @Component({
   selector: "transaction-log",
   templateUrl: "transaction-log.component.html",
 })
```

Comment:
Nice use of components. This will clean things up then other transaction logs are added.